src/main/java/me/arminb/sara/services/UserServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package me.arminb.sara.services;
+
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import me.arminb.sara.entities.User;
+import org.bson.Document;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import me.arminb.sara.dao.UserDAO;
+import me.arminb.sara.entities.User;
+
+import java.util.List;
+
+
+@Service("userService")
+public class UserServiceImpl implements UserService {
+
+    @Autowired
+    private UserDAO userDAO;
+
+    @Autowired
+    private MongoDatabase database;
+
+    @Override
+    public List<User> findAll() {
+        return userDAO.findAll();
+    }
+
+    @Override
```

Comment:
We will add security checks in this layer later. For now, it is just delegating work to the DAO layer which is fine. After you implemented exception translation in the DAO layer, you need to decide if you want to rethrow them to the above layer here or translate them to serivce layer-specific exceptions and then throw them up.